sbt-utils.js

```diff
@@ -163,3 +213,102 @@ const sbtPluginsPath = function (projectPath) {
   return [];
 };
 exports.parseKVDep = parseKVDep;
+
+const sbtInvoker = function (debugMode, path, tempSbtPlugins) {
+  let SBT_CMD = process.env.SBT_CMD || "sbt";
+  let sbtVersion = determineSbtVersion(path);
+  if (debugMode) {
+    console.log("Detected sbt version: " + sbtVersion);
+  }
+  const standalonePluginFileSupported = standalonePluginFile(sbtVersion);
+
+  let enablePluginString =  isDependencyTreeBuiltIn(sbtVersion) ? ENABLE_SBT_PLUGIN : ADD_SBT_PLUGIN;
+
+  if (standalonePluginFileSupported) {
+    fs.writeFileSync(tempSbtPlugins, enablePluginString);
+  }
+
+  return {
+    invokeDependencyList: function(commandPrefix, basePath, timeoutMs) {
+      let dependencyListCmd;
+      
+      let tmpDir = pathLib.join(os.tmpdir(), 'cdxgen-sbt-');
```

Comment:
This causes an issues on MacOS, it's indeterministic. Didn't have time to check if it's related to e.g. https://github.com/nodejs/node/issues/11422

Reply:
Meaning it fails on MacOS, or just returns random paths? Either way, would be better if it also works on Mac OS :)